### PR TITLE
Improvements to card UX

### PR DESCRIPTION
**Pull request by harish86**

### CODE REVIEW

File: web/static/components/glmodal.vue [hunk 1]
```diff
@@ -54,6 +56,8 @@ export default {
       vm.windowClass = options['class'];
       vm.options = options;
       vm.open = true;
+
+      console.log(vm.modalData);
```
Comment (parallel588) [inline web/static/components/glmodal.vue]: Hi @harish86, I think we need to remove debug info.

File: web/static/components/cards/show.vue [hunk 2]
```diff
@@ -0,0 +1,426 @@
+<template>
+  <div class="card-show">
+    <div class="row">
+      <div class="col-lg-9 card-info">
+        <div class="card-title">
+          <i class="fa fa-rocket" style="color:#d8d8d8;"></i>
```
Comment (parallel588) [inline web/static/components/cards/show.vue]: Hi @harish86, may be better to clear html and move all styles to css.

File: web/static/components/cards/show.vue [hunk 3]
```diff
@@ -0,0 +1,426 @@
...
+          </el-date-picker>
+        </div>
+
+        <div class="card-description">
+          <markdown-text-edit v-model="card.description" v-on:input="updateCard" placeholder="Write a description" ></markdown-text-edit>
+        </div>
+
+        <div class="card-contacts-container">
+          <div class="title">
+            <i class="fa fa-user" style="color:#d8d8d8;"></i> Contacts
```
Comment (parallel588) [inline web/static/components/cards/show.vue]: Hi @harish86, may be better to clear html and move all styles to css.

File: web/static/components/cards/show.vue (hunk 2 above)
Comment (harish86) [inline web/static/components/cards/show.vue]: @parallel588 ok

File: web/static/components/glmodal.vue (hunk 1 above)
Comment (harish86) [inline web/static/components/glmodal.vue]: @parallel588 ok

File: web/static/components/cards/show.vue (hunk 3 above)
Comment (harish86) [inline web/static/components/cards/show.vue]: @parallel588 agree

**Comment by antoineherzog** [comment]
@harish86 please make sure to have the precommit checker activated 

/Users/garie/cercle_phoenix/web/static/components/cards/show.vue
  195:9   error  Identifier 'contact_ids' is not in camel case  camelcase
  207:7   error  Identifier 'card_id' is not in camel case      camelcase
  236:25  error  Strings must use singlequote                   quotes
  301:21  error  Identifier 'contact_ids' is not in camel case  camelcase
  370:40  error  Expected '===' and instead saw '=='            eqeqeq
  394:16  error  Identifier 'contact_ids' is not in camel case  camelcase
  394:45  error  Identifier 'contact_ids' is not in camel case  camelcase

**Pull request merged**